test/functional/query-runner/implicit-results.ts

```diff
@@ -0,0 +1,57 @@
+import "reflect-metadata";
+import { Connection } from "../../../src";
+import { closeTestingConnections, createTestingConnections } from "../../utils/test-utils";
+import { expect } from "chai";
+
+describe.only("query runner > implicit results", () => {
```

Comment:
`.only` - I end up accidentally leaving this around as well!  Can you remove it for merge, though?

```suggestion
describe("query runner > implicit results", () => {
```